src/components/Protein/Summary/index.tsx

```diff
@@ -0,0 +1,336 @@
+                    <IsoformSelector />
+                    {renderComparisonButton ? (
+                      <FullScreenButton
+                        element={comparisonContainerRef.current}
+                        className={css(
+                          'vf-button',
+                          'vf-button--secondary',
+                          'vf-button--sm',
+                          'comparison-button',
+                          'icon',
+                          'icon-common',
+                          { disabled: !isoform }
+                        )}
+                        disabled={!isoform}
+                        dataIcon={'\uF0DB'}
+                        tooltip="View in comparison mode"
+                        onFullScreenHook={() => setComparisonMode(true)}
+                        onExitFullScreenHook={() => setComparisonMode(false)}
+                      />
+                    ) : null}
+                  </td>
+                </tr>
+              ) : null}
+            </tbody>
+          </table>
+        </div>
+        <div className={css('vf-stack')}>
+          <section>
+            <section>
+              <h5>External Links</h5>
+              <ul className={css('no-bullet')}>
+                <li>
+                  <UniProtLink id={metadata.accession} className={css('ext')}>
+                    View {metadata.accession} in UniProtKB
+                  </UniProtLink>
+                </li>
+              </ul>
+            </section>
+            <hr style={{ margin: '0.8em' }} />
+            {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
+            <label>
+              <h5>Export Matches [TSV]</h5>
+              <FileExporter
+                description={{
+                  main: { key: 'protein' },
+                  protein: {
+                    db: metadata.source_database,
+                    accession: metadata.accession,
+                  },
+                  entry: { integration: 'all' },
+                }}
+                count={metadata.counters.entries as number}
+                fileType="tsv"
+                primary="entry"
+                secondary="protein"
+                minWidth={minWidth}
+              />
+            </label>
+            <hr style={{ margin: '0.8em' }} />
+            {/* <HmmerButton
+              sequence={metadata.sequence}
+              accession={metadata.accession}
+              title="Search protein with HMMER"
+              minWidth={minWidth}
+            />
+            <br /> */}
+            <IPScanButton
+              sequence={splitSequenceByChunks(
+                metadata.sequence,
+                metadata.id || ''
+              )}
+              title="Search protein with InterProScan"
+              minWidth={minWidth}
+            />
+            <DownloadButton
+              sequence={metadata.sequence}
+              accession={metadata.accession}
+            />
```

Comment:
Suggestions:

- remove the `<h5>` element above the TSV export button, and use a custom label for the button's title (requires changes I suggested to the `FileExporter` and `File` components.
- reorganize the buttons (makes more sense to have to two download buttons together) 
- change the title of the `IPSCanButton` from "Search protein with InterProScan" to "Search sequence with InterProScan"

```suggestion
            {/* <HmmerButton
              sequence={metadata.sequence}
              accession={metadata.accession}
              title="Search protein with HMMER"
              minWidth={minWidth}
            />
            <br /> */}
            <IPScanButton
              sequence={splitSequenceByChunks(
                metadata.sequence,
                metadata.id || ''
              )}
              title="Search sequence with InterProScan"
              minWidth={minWidth}
            />
            {/* eslint-disable-next-line jsx-a11y/label-has-associated-control */}
            <label>
              <FileExporter
                description={{
                  main: { key: 'protein' },
                  protein: {
                    db: metadata.source_database,
                    accession: metadata.accession,
                  },
                  entry: { integration: 'all' },
                }}
                count={metadata.counters.entries as number}
                fileType="tsv"
                primary="entry"
                secondary="protein"
                minWidth={minWidth}
                label="Download matches (TSV)"
              />
            </label>
            <DownloadButton
              sequence={metadata.sequence}
              accession={metadata.accession}
            />
```